channels/us.m3u

```diff
@@ -1681,8 +1677,6 @@ https://content.uplynk.com/channel/36953f5b6546464590d2fcd954bc89cf.m3u8
 https://2-fss-1.streamhoster.com/pl_122/201794-1414514-1/playlist.m3u8
 #EXTINF:-1 tvg-id="VictoryTelevisionNetwork.us" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/yqA2PJj.jpg" group-title="Religious",Victory Television Network (240p)
 http://184.173.179.163:1935/victorytelevisionnetwork/victorytelevisionnetwork/playlist.m3u8
-#EXTINF:-1 tvg-id="VidaMejorTV.us" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/W5C1ZRd.png" group-title="",Vida Mejor TV (480p)
-https://uni10rtmp.tulix.tv/bettervida/bettervida.stream/chunklist.m3u8
```

Comment:
VidaMejorTV is 3ABN Latino, right?

Reply:
We already have 3ABN Latino, which is why I've deleted this stream.

Reply:
No problem.